be/src/exec/pipeline/sort/local_partition_topn_context.cpp

```diff
@@ -150,11 +152,18 @@ LocalPartitionTopnContext* LocalPartitionTopnContextFactory::create(int32_t driv
 
     if (_ctxs[driver_sequence] == nullptr) {
         _ctxs[driver_sequence] = std::make_shared<LocalPartitionTopnContext>(
-                _t_partition_exprs, _sort_exec_exprs, _is_asc_order, _is_null_first, _sort_keys, _offset,
-                _partition_limit, _topn_type, _order_by_types, _materialized_tuple_desc, _parent_node_row_desc,
+                _t_partition_exprs, _sort_exprs, _is_asc_order, _is_null_first, _sort_keys, _offset, _partition_limit,
+                _topn_type, _order_by_types, _materialized_tuple_desc, _parent_node_row_desc,
                 _parent_node_child_row_desc);
     }
 
     return _ctxs[driver_sequence].get();
 }
+
+Status LocalPartitionTopnContextFactory::prepare(RuntimeState* state) {
+    RETURN_IF_ERROR(Expr::prepare(_sort_exprs, state));
+    RETURN_IF_ERROR(Expr::open(_sort_exprs, state));
+    return Status::OK();
+}
```

Comment:
seem not need this, because _sort_exec_exprs.lhs_ordering_expr_ctxs() is prepared and opened in _sort_exec_exprs's prepare() and open()

Reply:
```
*** Check failure stack trace: ***
    @          0xe6532ad  google::LogMessage::Fail()
    @          0xe65571f  google::LogMessage::SendToLog()
    @          0xe652dfe  google::LogMessage::Flush()
    @          0xe655d29  google::LogMessageFatal::~LogMessageFatal()
    @          0xbe5031a  starrocks::ExprContext::evaluate()
    @          0x9c25f57  _ZZN9starrocks10vectorized11DataSegment4initEPKSt6vectorIPNS_11ExprContextESaIS4_EERKSt10shared_ptrINS0_5ChunkEEENKUlS4_PNS_4ExprEPSA_E_clES4_SF_SG_
    @          0x9c262b9  starrocks::vectorized::DataSegment::init()
    @          0x9c365a4  starrocks::vectorized::DataSegment::DataSegment()
    @          0x9c52809  __gnu_cxx::new_allocator<>::construct<>()
    @          0x9c52402  std::allocator_traits<>::construct<>()
    @          0x9c5ecf3  std::vector<>::emplace_back<>()
    @          0x9c5659b  starrocks::vectorized::ChunksSorterTopn::_build_sorting_data()
    @          0x9c55e1b  starrocks::vectorized::ChunksSorterTopn::_sort_chunks()
    @          0x9c5506b  starrocks::vectorized::ChunksSorterTopn::done()
```